I_dont_have_vinigret.py

```diff
@@ -0,0 +1,38 @@
+import random
+import time
+import datetime
+
+INPUT_DATE_MESSAGE = "Please enter date in YYYY-MM-DD format: "
+MONDAY_MESSAGE = "I don't have vinaigrette!"
+
+
+# the main function - read the data and call the other functions to calculate
+# the random date and check if the new date is in monday
+def print_date():
+    start_date = input(INPUT_DATE_MESSAGE)
+    end_date = input(INPUT_DATE_MESSAGE)
+    rand_date = random_date(start_date, end_date, '%Y-%m-%d', random.random())
+    print(rand_date)
+    if check_if_monday(rand_date):
+        print(MONDAY_MESSAGE)
+
+
+# function to create the new date
```

Comment:
Please add the documentation after the function definition.
Also, it should be more detailed and to describe the parameters it gets and the value it returns (if so).

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/159840460-7facc925-50b3-4242-898f-f3b59c00e526.JPG)
It should look like this